interview-questions.md

```diff
@@ -6,46 +6,47 @@ First, without external resources. Challenge yourself to answer from memory.
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely on your own, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-1. What is a gem?
+1. What is a gem? 
 
-Your answer:
+Your answer: I believe that a gem a big storage that contains standard coding functionality that makes it easier to use the functionality to an app.
 
-Researched answer:
+Researched answer: gems are stored and managed in the package manager named RubyGems, it also is easily accessible for all developers to sharing them to one and another.
 
 2. What does the acronym CRUD stand for?
 
-Your answer:
+Your answer: Create, read, update, and delete. CRUD lets you sort through data bases
 
-Researched answer:
+Researched answer: CRUD has all the functions that are required to interact with relational databases.
```

Comment:
There is a lot of room for expansion on these answers.  Keep in mind that short concise answers will likely not impress an employer in an interview.